incubator/open-science-ce/open-science-ce/templates/deployment.yaml

```diff
@@ -122,7 +119,7 @@ spec:
       {{ end }}
       - name: open-science-ce
         image: opensciencegrid/htcondor-ce:fresh
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
Could you not make this change until we're done testing/debugging/whatever? It's hard to get Kubernetes to update the image otherwise.

Reply:
We can put it back to Always. That is the default mode for SLATE. What ever works best for your development.

Reply:
The init containers above have `imagePullPolicy: Always` for both the CE and submit containers. This change just prevents us from pulling the CE image down twice during a deployment